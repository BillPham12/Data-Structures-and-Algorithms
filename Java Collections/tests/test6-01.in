1
123
123
12312312
312312312312
312
3121
1
1

1
1
123
213
123
123
12
