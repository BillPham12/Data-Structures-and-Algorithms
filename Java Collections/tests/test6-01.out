
1
12
123
213
312
3121
12312312
312312312312
